### PR TITLE
Feature/ben

### PR DESCRIPTION
**Pull request by ghost**
Merging a test to match Code and see if it works.

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "react-scripts": "5.0.0",
     "web-vitals": "^2.1.2"
   },
-  "proxy": "http://localhost:3003",
+  "proxy": "http://localhost:3001",
```
Comment (ghost) [inline client/package.json:19]: this one

File: client/public/index.html [hunk 2]
```diff
@@ -25,7 +25,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Deep Thoughts</title>
+    <title>What's Happenin'</title>
```
Comment (ghost) [inline client/public/index.html:28]: this one

File: client/src/components/ThoughtForm/index.js [hunk 3]
```diff
@@ -61,7 +61,7 @@ const ThoughtForm = () => {
             <form className="flex-row justify-center justify-space-between-md align-stretch"
                 onSubmit={handleFormSubmit} >
             <textarea
-            placeholder="Here's a new thought..."
+            placeholder="Tell people what's happenin'..."
```
Comment (ghost) [inline client/src/components/ThoughtForm/index.js:64]: this one

File: client/src/components/ThoughtList/index.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ import { Link } from 'react-router-dom'
 
 const ThoughtList = ({ thoughts, title }) => {
   if (!thoughts.length) {
-    return <h3>No Thoughts Yet</h3>;
+    return <h3>No Events Yet</h3>;
```
Comment (ghost) [inline client/src/components/ThoughtList/index.js:6]: this one

File: server/server.js [hunk 5]
```diff
@@ -6,7 +6,7 @@ const {typeDefs, resolvers} = require('./schemas');
 const {authMiddleware} = require('./utils/auth');
 const db = require('./config/connection');
 
-const PORT = process.env.PORT || 3003;
```
Comment (ghost) [inline server/server.js:9]: this one

File: src/components/ThoughtForm/index.js [hunk 6]
```diff
@@ -61,7 +61,7 @@ const ThoughtForm = () => {
             <form className="flex-row justify-center justify-space-between-md align-stretch"
                 onSubmit={handleFormSubmit} >
             <textarea
-            placeholder="Here's a new thought..."
+            placeholder="Tell people what's happenin'..."
```
Comment (ghost) [inline src/components/ThoughtForm/index.js:64]: this one

**Review by ghost — COMMENTED** [review]
test

**Review by BTantillo — APPROVED** [review]
Testing merging

**Pull request merged**